ci/pipeline.yml

```
@@ -202,10 +206,10 @@ jobs:
           DB: mysql
           RAKE_TASK: spec:unit:director
 
-  - name: unit-director-postgres-13
+  - name: unit-director-postgres-17
     public: true
     serial: true
-    serial_groups: [ unit-director-postgres-13 ]
+    serial_groups: [ unit-director-postgres-17 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)